### ISSUE TITLE
Invalid RCTFontStyle

### ISSUE THREAD

**Issue by togenius1** [Abhinandan-Kushwaha/react-native-gifted-charts#254]
<img width="447" alt="Screen Shot 2565-07-24 at 20 28 05" src="https://user-images.githubusercontent.com/90688382/180649171-78850990-0383-4b1b-837d-be4d12108e73.png">

ERROR on ios but still working in Android simulator.
Please see below

 ERROR  Warning: Each child in a list should have a unique "key" prop.

Check the render method of `PieChartMain`. See https://reactjs.org/link/warning-keys for more information.
    in PieChartMain (at PieChart/index.tsx:138)
    in RCTView (at View.js:32)
    in View (at PieChart/index.tsx:137)
    in RCTView (at View.js:32)
    in View (at PieChart/index.tsx:96)
    in PieChart (at ExpensesScreen.tsx:55)
    in RCTView (at View.js:32)
    in View (at ExpensesScreen.tsx:54)
    in RecentExpenses (at SceneView.tsx:132)
    in StaticContainer
    in EnsureSingleNavigator (at SceneView.tsx:124)
    in SceneView (at useDescriptors.tsx:217)
    in RCTView (at View.js:32)
    in View (at Screen.tsx:63)
    in RCTView (at View.js:32)
    in View (at Background.tsx:13)
    in Background (at Screen.tsx:58)
    in Screen (at BottomTabView.tsx:130)
    in RNSScreen (at createAnimatedComponent.js:211)
    in AnimatedComponent (at createAnimatedComponent.js:264)
    in AnimatedComponentWrapper (at src/index.native.tsx:257)
    in MaybeFreeze (at src/index.native.tsx:256)
    in Screen (at ScreenFallback.tsx:38)
    in MaybeScreen (at BottomTabView.tsx:123)
    in RNSScreenNavigationContainer (at src/index.native.tsx:330)
    in ScreenContainer (at ScreenFallback.tsx:29)
    in MaybeScreenContainer (at BottomTabView.tsx:93)
    in RCTView (at View.js:32)
    in View (at SafeAreaProviderCompat.tsx:42)
    in SafeAreaProviderCompat (at BottomTabView.tsx:92)
    in BottomTabView (at createBottomTabNavigator.tsx:118)
    in Unknown (at createBottomTabNavigator.tsx:117)
    in BottomTabNavigator (at App.tsx:27)
    in ExpensesOverview (at SceneView.tsx:132)
    in StaticContainer
    in EnsureSingleNavigator (at SceneView.tsx:124)
    in SceneView (at useDescriptors.tsx:217)
    in RCTView (at View.js:32)
    in View (at DebugContainer.native.tsx:27)
    in DebugContainer (at NativeStackView.native.tsx:76)
    in MaybeNestedStack (at NativeStackView.native.tsx:288)
    in RNSScreen (at createAnimatedComponent.js:211)
    in AnimatedComponent (at createAnimatedComponent.js:264)
    in AnimatedComponentWrapper (at src/index.native.tsx:257)
    in MaybeFreeze (at src/index.native.tsx:256)
    in Screen (at NativeStackView.native.tsx:214)
    in SceneView (at NativeStackView.native.tsx:342)
    in RNSScreenStack (at src/index.native.tsx:188)
    in ScreenStack (at NativeStackView.native.tsx:331)
    in NativeStackViewInner (at NativeStackView.native.tsx:388)
    in RNCSafeAreaProvider (at SafeAreaContext.tsx:87)
    in SafeAreaProvider (at SafeAreaProviderCompat.tsx:46)
    in SafeAreaProviderCompat (at NativeStackView.native.tsx:387)
    in NativeStackView (at createNativeStackNavigator.tsx:72)
    in Unknown (at createNativeStackNavigator.tsx:71)
    in NativeStackNavigator (at App.tsx:112)
    in EnsureSingleNavigator (at BaseNavigationContainer.tsx:430)
    in BaseNavigationContainer (at NavigationContainer.tsx:132)
    in ThemeProvider (at NavigationContainer.tsx:131)
    in NavigationContainerInner (at App.tsx:111)
    in App (at ExpenseTrackerWithLocalStorage/index.js:14)
    in PersistGate (at ExpenseTrackerWithLocalStorage/index.js:13)
    in Provider (at ExpenseTrackerWithLocalStorage/index.js:12)
    in Root (at renderApplication.js:50)
    in RCTView (at View.js:32)
    in View (at AppContainer.js:92)
    in RCTView (at View.js:32)
    in View (at AppContainer.js:119)
    in AppContainer (at renderApplication.js:43)
    in ExpenseTrackerWithLocalStorage(RootComponent) (at renderApplication.js:60)

**Comment by togenius1** [comment]
Done

**Comment by belal-bh** [comment]
Hi, I am facing same issue.

<img width="202" alt="image" src="https://user-images.githubusercontent.com/30633331/215084711-38ec7365-6684-4fc3-a3e5-bc05e380e94d.png">


```
 ERROR  Warning: Each child in a list should have a unique "key" prop.

Check the render method of `PieChartMain`. See https://reactjs.org/link/warning-keys for more information.
    in PieChartMain (created by PieChart)
    in RCTView (created by View)
    in View (created by PieChart)
    in RCTView (created by View)
    in View (created by PieChart)
    in PieChart (created by TransactionPieChart)
    in RCTView (created by View)
    in View (created by TransactionPieChart)
    in TransactionPieChart (created by TransactionInsightsScreen)
    in RCTView (created by View)
    in View (created by TransactionInsightsScreen)
    in RCTView (created by View)
    in View (created by TransactionInsightsScreen)
    in RCTView (created by View)
    in View (created by ScrollView)
    in RCTScrollView (created by ScrollView)
    in ScrollView (created by ScrollView)
    in ScrollView (created by TransactionInsightsScreen)
    in RCTView (created by View)
    in View (created by TransactionInsightsScreen)
    in TransactionInsightsScreen (created by SceneView)
    in StaticContainer
    in EnsureSingleNavigator (created by SceneView)
    in SceneView (created by SceneView)
    in RCTView (created by View)
    in View (created by DebugContainer)
    in DebugContainer (created by MaybeNestedStack)
    in MaybeNestedStack (created by SceneView)
    in RCTView (created by View)
    in View (created by SceneView)
    in RNSScreen (created by AnimatedComponent)
    in AnimatedComponent
    in AnimatedComponentWrapper (created by InnerScreen)
    in Suspender (created by Freeze)
    in Suspense (created by Freeze)
    in Freeze (created by DelayedFreeze)
    in DelayedFreeze (created by InnerScreen)
    in InnerScreen (created by Screen)
    in Screen (created by SceneView)
    in SceneView (created by NativeStackViewInner)
    in Suspender (created by Freeze)
    in Suspense (created by Freeze)
    in Freeze (created by DelayedFreeze)
    in DelayedFreeze (created by ScreenStack)
    in RNSScreenStack (created by ScreenStack)
    in ScreenStack (created by NativeStackViewInner)
    in NativeStackViewInner (created by NativeStackView)
    in RCTView (created by View)
    in View (created by SafeAreaInsetsContext)
    in SafeAreaProviderCompat (created by NativeStackView)
    in NativeStackView (created by NativeStackNavigator)
    in PreventRemoveProvider (created by NavigationContent)
    in NavigationContent
    in Unknown (created by NativeStackNavigator)
    in NativeStackNavigator (created by HomeNavigator)
    in HomeNavigator (created by SceneView)
    in StaticContainer
    in EnsureSingleNavigator (created by SceneView)
    in SceneView (created by BottomTabView)
    in RCTView (created by View)
    in View (created by Screen)
    in RCTView (created by View)
    in View (created by Background)
    in Background (created by Screen)
    in Screen (created by BottomTabView)
    in RNSScreen (created by AnimatedComponent)
    in AnimatedComponent
    in AnimatedComponentWrapper (created by InnerScreen)
    in Suspender (created by Freeze)
    in Suspense (created by Freeze)
    in Freeze (created by DelayedFreeze)
    in DelayedFreeze (created by InnerScreen)
    in InnerScreen (created by Screen)
    in Screen (created by MaybeScreen)
    in MaybeScreen (created by BottomTabView)
    in RNSScreenContainer (created by ScreenContainer)
    in ScreenContainer (created by MaybeScreenContainer)
    in MaybeScreenContainer (created by BottomTabView)
    in RCTView (created by View)
    in View (created by SafeAreaInsetsContext)
    in SafeAreaProviderCompat (created by BottomTabView)
    in BottomTabView (created by BottomTabNavigator)
    in PreventRemoveProvider (created by NavigationContent)
    in NavigationContent
    in Unknown (created by BottomTabNavigator)
    in BottomTabNavigator (created by MainNavigator)
    in MainNavigator (created by SceneView)
    in StaticContainer
    in EnsureSingleNavigator (created by SceneView)
    in SceneView (created by SceneView)
    in RCTView (created by View)
    in View (created by DebugContainer)
    in DebugContainer (created by MaybeNestedStack)
    in MaybeNestedStack (created by SceneView)
    in RCTView (created by View)
    in View (created by SceneView)
    in RNSScreen (created by AnimatedComponent)
    in AnimatedComponent
    in AnimatedComponentWrapper (created by InnerScreen)
    in Suspender (created by Freeze)
    in Suspense (created by Freeze)
    in Freeze (created by DelayedFreeze)
    in DelayedFreeze (created by InnerScreen)
    in InnerScreen (created by Screen)
    in Screen (created by SceneView)
    in SceneView (created by NativeStackViewInner)
    in Suspender (created by Freeze)
    in Suspense (created by Freeze)
    in Freeze (created by DelayedFreeze)
    in DelayedFreeze (created by ScreenStack)
    in RNSScreenStack (created by ScreenStack)
    in ScreenStack (created by NativeStackViewInner)
    in NativeStackViewInner (created by NativeStackView)
    in RCTView (created by View)
    in View (created by SafeAreaInsetsContext)
    in SafeAreaProviderCompat (created by NativeStackView)
    in NativeStackView (created by NativeStackNavigator)
    in PreventRemoveProvider (created by NavigationContent)
    in NavigationContent
    in Unknown (created by NativeStackNavigator)
    in NativeStackNavigator (created by Router)
    in EnsureSingleNavigator
    in BaseNavigationContainer
    in ThemeProvider
    in NavigationContainerInner (created by Router)
    in Router (created by App)
    in QueryClientProvider (created by App)
    in I18nextProvider (created by LanguageProvider)
    in LanguageProvider (created by App)
    in ThemeProvider (created by Provider)
    in RCTView (created by View)
    in View (created by Portal.Host)
    in Portal.Host (created by Provider)
    in RNCSafeAreaProvider (created by SafeAreaProvider)
    in SafeAreaProvider (created by SafeAreaInsetsContext)
    in SafeAreaProviderCompat (created by Provider)
    in Provider (created by ThemeProvider)
    in ThemeProvider (created by App)
    in GlobalStateProvider (created by App)
    in App
    in RCTView (created by View)
    in View (created by AppContainer)
    in RCTView (created by View)
    in View (created by AppContainer)
    in AppContainer
    in MoneyManager23(RootComponent)
```

Any solution ? @togenius1

**Comment by Abhinandan-Kushwaha** [comment]
This has been fixed in version `1.3.0`
